### DISCUSSION TITLE
Quais skills são necessárias para ser um bom profissional

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#221] [Discussão]
Gostaria de levantar esse ponto! Quais outras habilidades além das técnicas devemos cultivar para se tornar um profissional melhor?! 👍

**Comment by felipefialho** [comment]
Hey @felquis. Já terminou aquele artigo? hehe

**Comment by woliveiras** [comment]
No Carreira Front End também fiz um levantamento desse tipo @Cleysonlb http://bit.ly/carreira-front-end 😄

**Comment by vitocaio** [comment]
Li um artigo do @woliveiras sobre carreira front-end e os requisitos necessários para avançar de junior para pleno e de pleno para sênior, recomendo muito, da uma lida, me deu até uma motivada :)

**Comment by elfiservice** [comment]
Vale a pena da o Check aqui by Udacity    ✅ 
[Habilidades de um desenvolvedor front-end](https://br.udacity.com/checklist-desenvolvedor-front-end?utm_source=vero&utm_medium=email&utm_content=a&utm_campaign=2018-05-25_BR%20Ebook%20Checklist%20de%20front-end&utm_term=Newsletter&vero_id=9572678855&vero_conv=Z3_qSuIADu6XuicUr7QGYTFmricer7zFuSEeuWcIK9ypm1oKh8QgvkP7RZ2b8WFbEjsn_I6R6jnX0dro6zLHbwhrhnDjy4gj)

**Comment by luizbills** [comment]
Tirando os requesitos técnicos (alto conhecimento da(s) linguagem e uma boa capacidade de pensamento lógico), acho que uma coisa que separa os bons dos medianos é a capacidade de "se virar": saber/querer ler um código-fonte, aprender por conta própria, analisar/debugar um problema ect.

Vejo muito profissional que no primeiro probleminha trava e não consegue sair do lugar sem receber um suporte adequado. Ou pior, aquela pessoa que é um baita programador(a) só que não consegue fazer nada ser ter um bom exemplo pronto.

**Comment by JorgeSegtowich** [comment]
@Cleysonlb acredito que além dos requisitos técnicos exite também os sociais. De ter um bom relacionamento com os colegas de trabalho, se comunicar bem e saber como agir em certas situações dentro do ambiente de trabalho.
[]'s

**Comment by bernardodiasc** [comment]
> acho que uma coisa que separa os bons dos medianos é a capacidade de "se virar"

considero isso uma das coisas que definem um profissional sênior: nunca fica bloqueado.
